pycid/core/macid.py

```diff
@@ -24,7 +23,7 @@ class MACID(MACIDBase):
     def get_ne(self, solver: Optional[str] = "enumpure") -> List[List[StochasticFunctionCPD]]:
         """
         Return a list of Nash equilbiria in the MACID. By default, this finds all pure NE using the 'enumpure'
-        pygambit solver. Use the 'solver' argument to change this behaviour.
+        pygambit solver. Use the 'solver' argument to change this behavior.
```

Comment:
I assume this is so you're consistent with the unfortunate American spelling  `behavior' in PyGAMBIT?

Reply:
yeah sadly its probably better to stick to americani"z"ed spellings I think